newswires/app/db/FingerpostWireEntry.scala

```diff
@@ -99,7 +97,13 @@ object FingerpostWireEntry extends SQLSyntaxSupport[FingerpostWireEntry] {
       )
     ).flatten
 
-    val whereClause = whereClauses match {
+    val dataOnlyWhereClauses = List(
```

Comment:
It seems like a deliberate choice to construct this as a separate val before merging the two lists of clauses, but I can't work out why from the naming.. What am I missing?

Reply:
ahh, we don't pass the id based clauses on the keywordCounts bit, because keywords bit is all about the total for each keyword in the whole result set, so if we're polling (more #34 than this PR) we want those counts to be 'latest'